src/components/access_group/AccessGroupTerms.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <main class="group-terms">
+    <a
```

Comment:
trying to find out what's the up to date UX here. But I think this is just a link to the access group (independent from where a user ended up here). This would make thinks easier and this is just rotuer link to the access group.